### PR TITLE
(test): Unit tests for flexible scaling feature 

### PR DESCRIPTION
**Pull request by afreen23**
- include unit tests for the enablement logic
 - implements https://issues.redhat.com/browse/RHSTOR-1589

Screenshot from 2021-02-25 00-34-00

### CODE REVIEW

**Comment by afreen23** [comment]
This feature is removed from internal mode.
Testing on baremetal requires lso installation.
e2e tests are not possible.

File: frontend/packages/ceph-storage-plugin/src/__tests__/flexible-scaling.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+describe('isFlexibleScaling', () => {
+  it('return truthy when atleast 3 nodes and no AZ is passed', () => {
+    expect(isFlexibleScaling(4, 0)).toBe(true);
+  });
+  it('return truthy when atleast 3 nodes and 1 AZ is passed', () => {
+    expect(isFlexibleScaling(4, 1)).toBe(true);
+  });
+  it('return truthy when atleast 3 nodes and 2 AZ is passed', () => {
+    expect(isFlexibleScaling(4, 2)).toBe(true);
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/__tests__/flexible-scaling.spec.ts:11]: I'll rather test 3 & 4 nodes both.
```suggestion
    expect(isFlexibleScaling(3, 2)).toBe(true);
```

**Comment by afreen23** [comment]
Reopened the PR for adding unit tests since e2e not possible https://github.com/openshift/console/pull/7653#issuecomment-774910316

File: frontend/packages/ceph-storage-plugin/src/__tests__/flexible-scaling.spec.ts (hunk 1 above)
Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/__tests__/flexible-scaling.spec.ts]: sounds good

**Pull request merged**